src/scenes/Activities/scenes/EventHistory.js

```diff
@@ -307,8 +309,8 @@ export default class EventHistory extends Component {
     const isExpanded = this.state.expanded.includes(txid);
 
     result[0] = (
-      <TableRow key={txid} selected={isExpanded} onClick={this.handleClick(txid)} className={classes.clickToExpandRow} >
```

Comment:
don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

Reply:
comment above